bedrock/base/tests/test_middleware.py

```diff
@@ -10,6 +10,7 @@
 from django.test.utils import override_settings
 from django.urls import reverse
 
+import csp.constants as csp
```

Comment:
nit: this renamed import would bite me, I suspect

Reply:
yeah. Perhaps I should add an import in the root `__init__.py` that imports the constants so an `import csp` will have the constants available. Which might be a nice end-user interface too for the Django settings.

Reply:
Updated 👍